### ISSUE TITLE
Staff Page - Social Link & Row bugs

### ISSUE THREAD

**Issue by devanflaherty** [tylerbodway/crossroadschurch.net#5]
**Social Link Bug**
We discovered a bug on the staff page - So on each staffing item you’ll see that it is showing links for facebook and twitter, even if they do not have a facebook or twitter, and those fields are not required in the Staff posts, so they shouldn't appear on all staff posts. Also, it is displaying the facebook and twitter links of members that have included their pages, on all members that come after them, that have NOT included twitter/facebook links. It repeats the last person's page links until another person has twitter or facebook links included.

To see what I mean, if you go to the http://dev.crossroadschurch.net/staff/ , click on Daniel Bulek. He has both a twitter and facebook account linked to his staff page, so that’s great. But, if you back out and click on the next staff member, Nick Campfield (who does not have a facebook or twitter account linked) you will see that the twitter and facebook links on his page are automatically linking to Daniel's fb/twitter accounts.

Finally, if you click on the next member Ralph Cassel (who has included facebook and twitter information), it is displaying correctly again.

It continues this pattern on the next pages as well. Gennarino DeStefano has a twitter linked to his page, but it also is putting his twitter information on the members that come after him (Margaret, Brian, Troy and Don) then stops at Daniel Fusco because he has his accounts linked. 

So basically, we need to have the twitter and facebook links only appear on staff members pages that have included links to their social media account.

**Row Population Bug **
NEXT, we noticed that the grid doesn’t populate fully, so if it creates a 4 x 3 grid it populates all of the top 2 rows, but the bottom row only has 2 of the 4 positions populated. The rest are on the next page, and so forth. Thanks!!

**Comment by tylerbodway** [comment]
@devanflaherty this should be fixed with last 'master' merge

**Comment by tylerbodway** [comment]
1872d1b0d62a5e805006ac3335b4ff341ad923ae
